doc/source/running.rst

```diff
@@ -6,78 +6,45 @@ Backing Up and Restoring Data
 .. contents::
    :local:
 
-.. _pbm.running.backup-coordinator:
-
-Running the Backup Coordinator
+Running |pbm|
 ================================================================================
 
-The |bc| can be executed on any server since it does not need a connection to a
-|mongod| instance. One |bc| is needed per deployment.
+The |pbm.app| program can be run on any server since it does not need a connection to a
+|mongod| instance. One |pbm.app| is needed per deployment.
 
-To start the |bc| run |pbm-coordinator| as follows:
+You start |pbm.app| as follows:
 
 .. code-block:: bash
 
-   $ pbm-coordinator --work-dir=<directory to store metadata>
-n
+   $ pbm --work-dir=<directory to store metadata>
+
 If the |opt-work-dir| option is not specified, it will use the default
-|work-dir|.  By default, the |bc| listens for agents on port 10000.
+|work-dir|.  By default, |pbm.app| listens for agents on port 10000.
 
 .. _pbm.running.storage.setting-up:
 
 Setting Up Storages for Backup and Restore Operations
 --------------------------------------------------------------------------------
 
-For running the backup (`pbmctl run backup`) and restore (`pbmctl run restore`)
-operations, you need to set up a storage. This is a location known to |pbmctl|
+For running the backup (`pbm backup`) and restore (`pbm restore`)
+operations, you need to set up a storage. This is a location known to |pbm.app|
 where the backup data are stored. All storages must be listed in the
 |storages-yml| file. You can use the name of the storage without
 having to supply its connection parameters on the command line. 
 
 .. code-block:: bash
 
-   $ pbmctl run backup --description="PBM Backup" --storage=localfs
+   $ pbm store set storage-file.yml
```

Comment:
Yes to that new example.
But we don't need the "./" on the front right? The ordinary user will install it by package and it will be in their normal path.